core/src/components/title/title.ios.scss

```diff
@@ -40,16 +40,17 @@
 // --------------------------------------------------
 
 :host(.title-large) {
-  @include padding(0, 16px);
+  @include padding(2px, 12px, 4px, 12px);
   @include transform-origin(left, center);
 
-  bottom: 0;
+  position: static;
 
   align-items: flex-end;
 
   min-width: 100%;
 
-  font-size: 34px;
+  font-size: dynamic-font-min(1, 34px);
```

Comment:
This is going to set a minimum font size. Did you mean to use `dynamic-font-max` instead?